test/fetch/data-uri.js

```diff
@@ -177,3 +177,14 @@ test('too long base64 url', async (t) => {
     t.fail(`failed to fetch ${dataURL}`)
   }
 })
+
+test('https://domain.com#', (t) => {
+  const domain = 'https://domain.com#'
```

Comment:
```suggestion
  const domain = 'https://domain.com/#'
```

unless this is expected, and then there's a bug in url serializer?

Reply:
I think this is correct. In chrome, the result has `/` too.